packages/tokens/src/figma-tokens/semantic/link.json

```diff
@@ -0,0 +1,30 @@
+{
+  "link": {
+    "color": {
+      "value": "{primary.600}",
+      "type": "color"
+    },
+    "color-hover": {
+      "value": "{primary.900}",
+      "type": "color"
+    },
+    "color-active": {
+      "value": "{primary.950}",
+      "type": "color"
+    },
```

Comment:
I believe this should link to the button default/hover/active colors, right? @LeaLeRose

Reply:
It does. However, we do not have semantic tokens for the button as of current, therefore we use the primitives here.